moped-editor/src/components/GridTable/GridTableFilters.js

```diff
@@ -575,6 +599,7 @@ const GridTableFilters = ({
               <Hidden smDown>
                 <Grid item xs={12} md={1} style={{ textAlign: "center" }}>
                   <IconButton
+                    disabled={!filterParameters[filterId].value}
```

Comment:
There's a small bug here, because you can end up in this weird state where you cannot delete a filter without filling it out.

<img width="1161" alt="Screen Shot 2022-10-05 at 5 37 50 PM" src="https://user-images.githubusercontent.com/14793120/194170340-6a860462-c65f-4404-91cf-da2d878890d0.png">

I think the disabled condition we want to enforce is when there is currently only one filter visible. So maybe testing on `Object.keys(filterParameters).length` ?